job/job.go

```diff
@@ -301,3 +301,10 @@ func unitPrintf(s string, nu unit.UnitNameInfo) (out string) {
 	out = strings.Replace(out, "%i", nu.Instance, -1)
 	return
 }
+
+// isTruthyValue returns true if a given string is any of "truthy" value,
+// i.e. "true", "yes", "1", "on", or "t".
+func isTruthyValue(astr string) bool {
```

Comment:
Could you add a test table for this function please?
also s/astr/s/, prefer simpler variable names in go